js/web/script/build.ts

```diff
@@ -416,6 +417,16 @@ async function main() {
         'BUILD_DEFS.DISABLE_WASM_THREAD': 'true',
       },
     });
+    // ort.training.wasm.min.js
+    await addAllWebBuildTasks({
+      outputBundleName: 'ort.training.wasm.min',
```

Comment:
should be "ort.training.wasm". could you also help to fix line 411? shoud remove ".min"